app/views/welcome/show.html.erb

```diff
@@ -14,11 +13,22 @@
           <li>Announce packages delivered to the space</li>
         </ul>
         <p>The sky is the limit!</p>
-        <p><%= link_to 'Get Started now', sign_in_path, class: 'btn btn--full btn--hl btn--biggest' %></p>
+        <p><%= link_to 'Get started', sign_in_path, class: 'btn btn--full btn--hl btn--biggest' %></p>
```

Comment:
Detail convention: We use title case on Buttons, so it should be `Get Started`

Reply:
Fixed!